api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTests.java

```diff
@@ -121,6 +122,38 @@ void shouldGetClaimById() {
         assertThat(claimResponse.getClaimant().getAddress()).isEqualToComparingFieldByField(claim.getClaimant().getAddress());
     }
 
+    @Test
+    void shouldGetAllClaims() {
+        // Given
+        Claim homerClaim = aValidClaimWithNinoAndRefernce(HOMER_NINO, HOMER_CLAIM_REFERENCE);
+        claimRepository.saveAll(List.of(homerClaim));
```

Comment:
if we are saving only one claim then may be just .save should do instead of saveAll or pass more than 1 claim